dashboard/src/modules/components/TableOfContent/index.jsx

```diff
@@ -176,29 +178,29 @@ const TableOfContent = () => {
     );
   };
   const getMyBreadCrumbClick = () => {
-    drillOut("rootMenu", "group:start_rollout", null);
+    drillOut("rootMenu", "group:start_rollout", initialBreadcrumb([]));
     setStack(1);
     setTableData(stack[0].files);
+    dispatch(updateContentData(stack[0]));
     setSearchSpace(stack[0].files);
-    setBreadCrumb(initialBreadcrumb([]));
     setParam("");
     setBreadCrumbLabels([]);
   };
   const getSubFolderData = (data) => {
     dispatch(fetchTOC(params["dataset_id"], data, true));
   };
   const attachBreadCrumbs = (data, firstHierarchyLevel) => {
-    breadCrumbLabels.push(data);
+    breadCrumbLabels.push(data.name);
```

Comment:
This doesn't seem to be working; or at least not as I'd expect.

We start out with "Root >", and clicking on "1-rw-4GiB" changes the breadcrumb to "Root > 1-rw-4GiB" as I'd expect ... but clicking on "sample1" doesn't change the breadcrumb display again, nor does clicking on "csv" inside that: it always stays "Root > 1-rw-4GiB" until I click on "Root" to reset.

Also, I'd expected that each breadcrumb would be an active link, but only "Root" is a link. For example, I'd expect to see "Root > 1-rw-4GiB > sample1 > csv", and that I could click on "Root", "1-rw-4GiB", or "sample1" to get back to that directory.

Reply:
Fair enough. I created [PBENCH-1242](https://issues.redhat.com/browse/PBENCH-1242) to track it separately.